feed_data.py

```diff
@@ -26,10 +29,33 @@ def analyze_keywords(text):
         lower_text = text.lower()
 
         # 🔴 PRIORITIZE NEGATIVE WORDS OVER POSITIVE IN MIXED CASES
-        negative_words = ["violence", "conflict", "death", "crisis", "humiliation",
-                          "deadly", "attack", "assault", "killings", "murder", "disaster",
-                          "scary", "danger", "terror", "threat", "catastrophe", "fatal"]
-        positive_words = ["growth", "success", "progress", "peace", "achievement", "hope"]
+        negative_words = [
```

Comment:
dont be confused. look at the commits. just formatted the code. the real changes are in sentiment_api.py